browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -85,6 +86,7 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
                          brave_wallet::mojom::kP3ACountTestNetworksSwitch));
   content::URLDataSource::Add(profile,
                               std::make_unique<SanitizedImageSource>(profile));
+  content::URLDataSource::Add(profile, std::make_unique<ThemeSource>(profile));
```

Comment:
That's an odd thing but chromium also does it. In fact each themed WebUI adds a new ThemeSource to profile replacing previous one, but they are like "equal" objects (per profile and trusted/untrusted). There should be only one ThemeSource (actually two - for untrusted) created but not dozens of them. 
And that amount of unneeded(but actually small) work is done on startup.

Reply:
Most likely the way that is implemented is just to avoid creation of ThemeSource at all if there is no web UI created that would need it.